
    <!doctype html>
    <html>
      <head>
          <meta http-equiv="Content-type" content="text/html; charset=utf-8">
          <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.2/jquery.min.js">
          </script>
          <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.16/css/jquery.dataTables.css">
          <script type="text/javascript" src="https://cdn.datatables.net/1.10.16/js/jquery.dataTables.js"></script>
          <style>
          # table {
              height: 250px;
              overflow-y:scroll;
          }
          </style>
      </head>
      <body><table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>downloaded</th>
      <th>htmlmade</th>
      <th>id</th>
      <th>source</th>
      <th>pmid</th>
      <th>pmcid</th>
      <th>fullTextIdList</th>
      <th>doi</th>
      <th>title</th>
      <th>authorString</th>
      <th>authorList</th>
      <th>authorIdList</th>
      <th>dataLinksTagsList</th>
      <th>journalInfo</th>
      <th>pubYear</th>
      <th>pageInfo</th>
      <th>abstractText</th>
      <th>affiliation</th>
      <th>publicationStatus</th>
      <th>language</th>
      <th>pubModel</th>
      <th>pubTypeList</th>
      <th>fullTextUrlList</th>
      <th>isOpenAccess</th>
      <th>inEPMC</th>
      <th>inPMC</th>
      <th>hasPDF</th>
      <th>hasBook</th>
      <th>hasSuppl</th>
      <th>citedByCount</th>
      <th>hasData</th>
      <th>hasReferences</th>
      <th>hasTextMinedTerms</th>
      <th>hasDbCrossReferences</th>
      <th>hasLabsLinks</th>
      <th>license</th>
      <th>hasEvaluations</th>
      <th>authMan</th>
      <th>epmcAuthMan</th>
      <th>nihAuthMan</th>
      <th>hasTMAccessionNumbers</th>
      <th>tmAccessionTypeList</th>
      <th>dateOfCompletion</th>
      <th>dateOfCreation</th>
      <th>firstIndexDate</th>
      <th>fullTextReceivedDate</th>
      <th>dateOfRevision</th>
      <th>electronicPublicationDate</th>
      <th>firstPublicationDate</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>PMC3579732</th>
      <td>True</td>
      <td>False</td>
      <td>23244533</td>
      <td>MED</td>
      <td>23244533</td>
      <td>PMC3579732</td>
      <td>{'fullTextId': 'PMC3579732'}</td>
      <td>10.1186/2041-1480-3-9</td>
      <td>Semantically enabling a genome-wide association study database.</td>
      <td>Beck T, Free RC, Thorisson GA, Brookes AJ.</td>
      <td>{'author': [{'fullName': 'Beck T', 'firstName': 'Tim', 'lastName': 'Beck', 'initials': 'T', 'authorId': OrderedDict([('@type', 'ORCID'), ('#text', '0000-0002-0292-7972')]), 'authorAffiliationDetailsList': OrderedDict([('authorAffiliation', OrderedDict([('affiliation', 'Department of Genetics, University of Leicester, University Road, Leicester, UK. tb143@leicester.ac.uk.')]))])}, {'fullName': 'Free RC', 'firstName': 'Robert C', 'lastName': 'Free', 'initials': 'RC', 'authorId': OrderedDict([('@type', 'ORCID'), ('#text', '0000-0001-8053-435X')])}, {'fullName': 'Thorisson GA', 'firstName': 'Gudmundur A', 'lastName': 'Thorisson', 'initials': 'GA', 'authorId': OrderedDict([('@type', 'ORCID'), ('#text', '0000-0001-5635-1860')])}, {'fullName': 'Brookes AJ', 'firstName': 'Anthony J', 'lastName': 'Brookes', 'initials': 'AJ'}]}</td>
      <td>{'authorId': [{'@type': 'ORCID', '#text': '0000-0001-5635-1860'}, {'@type': 'ORCID', '#text': '0000-0001-8053-435X'}, {'@type': 'ORCID', '#text': '0000-0002-0292-7972'}]}</td>
      <td>{'dataLinkstag': ['altmetrics', 'supporting_data']}</td>
      <td>{'issue': '1', 'volume': '3', 'journalIssueId': '1993738', 'dateOfPublication': '2012 Dec', 'monthOfPublication': '12', 'yearOfPublication': '2012', 'printPublicationDate': '2012-12-01', 'journal': {'title': 'Journal of biomedical semantics', 'ISOAbbreviation': 'J Biomed Semantics', 'medlineAbbreviation': 'J Biomed Semantics', 'NLMid': '101531992', 'ISSN': '2041-1480', 'ESSN': '2041-1480'}}</td>
      <td>2012</td>
      <td>9</td>
      <td><h4>Unlabelled</h4><h4>Background</h4>The amount of data generated from genome-wide association studies (GWAS) has grown rapidly, but considerations for GWAS phenotype data reuse and interchange have not kept pace. This impacts on the work of GWAS Central - a free and open access resource for the advanced querying and comparison of summary-level genetic association data. The benefits of employing ontologies for standardising and structuring data are widely accepted. The complex spectrum of observed human phenotypes (and traits), and the requirement for cross-species phenotype comparisons, calls for reflection on the most appropriate solution for the organisation of human phenotype data. The Semantic Web provides standards for the possibility of further integration of GWAS data and the ability to contribute to the web of Linked Data.<h4>Results</h4>A pragmatic consideration when applying phenotype ontologies to GWAS data is the ability to retrieve all data, at the most granular level possible, from querying a single ontology graph. We found the Medical Subject Headings (MeSH) terminology suitable for describing all traits (diseases and medical signs and symptoms) at various levels of granularity and the Human Phenotype Ontology (HPO) most suitable for describing phenotypic abnormalities (medical signs and symptoms) at the most granular level. Diseases within MeSH are mapped to HPO to infer the phenotypic abnormalities associated with diseases. Building on the rich semantic phenotype annotation layer, we are able to make cross-species phenotype comparisons and publish a core subset of GWAS data as RDF nanopublications.<h4>Conclusions</h4>We present a methodology for applying phenotype annotations to a comprehensive genome-wide association dataset and for ensuring compatibility with the Semantic Web. The annotations are used to assist with cross-species genotype and phenotype comparisons. However, further processing and deconstructions of terms may be required to facilitate automatic phenotype comparisons. The provision of GWAS nanopublications enables a new dimension for exploring GWAS data, by way of intrinsic links to related data resources within the Linked Data web. The value of such annotation and integration will grow as more biomedical resources adopt the standards of the Semantic Web.</td>
      <td>Department of Genetics, University of Leicester, University Road, Leicester, UK. tb143@leicester.ac.uk.</td>
      <td>epublish</td>
      <td>eng</td>
      <td>Electronic</td>
      <td>{'pubType': ['research-article', 'Journal Article']}</td>
      <td>{'fullTextUrl': [{'availability': 'Open access', 'availabilityCode': 'OA', 'documentStyle': 'pdf', 'site': 'Unpaywall', 'url': 'https://jbiomedsem.biomedcentral.com/counter/pdf/10.1186/2041-1480-3-9'}, {'availability': 'Free', 'availabilityCode': 'F', 'documentStyle': 'html', 'site': 'PubMedCentral', 'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/pmid/23244533/?tool=EBI'}, {'availability': 'Free', 'availabilityCode': 'F', 'documentStyle': 'pdf', 'site': 'PubMedCentral', 'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/pmid/23244533/pdf/?tool=EBI'}, {'availability': 'Subscription required', 'availabilityCode': 'S', 'documentStyle': 'doi', 'site': 'DOI', 'url': 'https://doi.org/10.1186/2041-1480-3-9'}, {'availability': 'Open access', 'availabilityCode': 'OA', 'documentStyle': 'html', 'site': 'Europe_PMC', 'url': 'https://europepmc.org/articles/PMC3579732'}, {'availability': 'Open access', 'availabilityCode': 'OA', 'documentStyle': 'pdf', 'site': 'Europe_PMC', 'url': 'https://europepmc.org/articles/PMC3579732?pdf=render'}]}</td>
      <td>Y</td>
      <td>Y</td>
      <td>Y</td>
      <td>Y</td>
      <td>N</td>
      <td>N</td>
      <td>5</td>
      <td>Y</td>
      <td>Y</td>
      <td>Y</td>
      <td>N</td>
      <td>Y</td>
      <td>cc by</td>
      <td>N</td>
      <td>N</td>
      <td>N</td>
      <td>N</td>
      <td>Y</td>
      <td>{'accessionType': 'refsnp'}</td>
      <td>2013-02-26</td>
      <td>2012-12-19</td>
      <td>2012-12-19</td>
      <td>2020-07-11</td>
      <td>2021-10-21</td>
      <td>2012-12-17</td>
      <td>2012-12-17</td>
    </tr>
  </tbody>
</table><script type="text/javascript">$(document).ready(function(){$('table').DataTable({
      "pageLength": 20
      });});</script>
      </body>
    </html>
    